common/Resources/ti.internal/kernel/module.js

```diff
@@ -0,0 +1,635 @@
+			this.wrapperCache[id] = wrapper;
+			return wrapper;
+		}
+
+		// See https://nodejs.org/api/modules.html#modules_all_together
+
+		/**
+		 * Require another module as a child of this module.
+		 * This parent module's path is used as the base for relative paths
+		 * when loading the child. Returns the exports object
+		 * of the child module.
+		 *
+		 * @param  {String} request  The path to the requested module
+		 * @return {Object}          The loaded module
+		 */
+		require (request) {
+			// 2. If X begins with './' or '/' or '../'
+			const start = request.substring(0, 2); // hack up the start of the string to check relative/absolute/"naked" module id
+			if (start === './' || start === '..') {
+				const loaded = this.loadAsFileOrDirectory(path.normalize(this.path + '/' + request));
+				if (loaded) {
+					return loaded.exports;
+				}
+			// Root/absolute path (internally when reading the file, we prepend "Resources/" as root dir)
+			} else if (request.substring(0, 1) === '/') {
+				const loaded = this.loadAsFileOrDirectory(path.normalize(request));
+				if (loaded) {
+					return loaded.exports;
+				}
+			} else {
+				// Despite being step 1 in Node.JS psuedo-code, we moved it down here because we don't allow native modules
+				// to start with './', '..' or '/' - so this avoids a lot of misses on requires starting that way
+
+				// 1. If X is a core module,
+				let loaded = this.loadCoreModule(request);
+				if (loaded) {
+					// a. return the core module
+					// b. STOP
+					return loaded;
+				}
+
+				// Look for CommonJS module
+				if (request.indexOf('/') === -1) {
+					// For CommonJS we need to look for module.id/module.id.js first...
+					const filename = `/${request}/${request}.js`;
+					// Only look for this _exact file_. DO NOT APPEND .js or .json to it!
+					if (this.filenameExists(filename)) {
+						loaded = this.loadJavascriptText(filename);
+						if (loaded) {
+							return loaded.exports;
+						}
+					}
+
+					// Then try module.id as directory
+					loaded = this.loadAsDirectory(`/${request}`);
+					if (loaded) {
+						return loaded.exports;
+					}
+				}
+
+				// Allow looking through node_modules
+				// 3. LOAD_NODE_MODULES(X, dirname(Y))
+				loaded = this.loadNodeModules(request, this.paths);
+				if (loaded) {
+					return loaded.exports;
+				}
+
+				// Fallback to old Titanium behavior of assuming it's actually an absolute path
+
+				// We'd like to warn users about legacy style require syntax so they can update, but the new syntax is not backwards compatible.
+				// So for now, let's just be quite about it. In future versions of the SDK (7.0?) we should warn (once 5.x is end of life so backwards compat is not necessary)
+				// eslint-disable-next-line max-len
+				console.warn(`require called with un-prefixed module id: ${request}, should be a core or CommonJS module. Falling back to old Ti behavior and assuming it's an absolute path: /${request}`);
```

Comment:
I assume you uncommented this on purpose? If so, then we should say something in the release notes.

Note that we see this warning on every app launch with "com.appcelerator.aca" if that module is not included. Most app developers are not using this module, so this is causing noise in the log.
https://github.com/appcelerator/titanium_mobile/blob/master/common/Resources/ti.internal/aca.js